katana/wapps/projects/suite_utils/verify_suite_file.py

```diff
@@ -0,0 +1,117 @@
+from collections import OrderedDict
```

Comment:
File name need to changed according to app.  verify_project_file.py  and also function within file need to be named correctly.